### PR TITLE
#1452 & #1359 HC EVAL - Fix typos for presumptive eligibility and add functionality to determine if case is a recertification

### PR DESCRIPTION
**Pull request by MDR22**
#1452 - Fixed a few typos related to presumptive eligibility

#1359 - Added functionality to navigate to STAT/REVW for CM + 1 through, and including, CM - 4 (but not past 07/23) to determine if a HC status code is present. For instances where the HC status code is present, script will default HC eval process to "Recertification".

### CODE REVIEW

File: notes/health-care-evaluation.vbs [hunk 1]
```diff
@@ -4925,6 +4985,7 @@ FOR i = 0 to total_clients
 				If HEALTH_CARE_MEMBERS(ref_numb_const, hc_memb) = HH_memb Then
 					HEALTH_CARE_MEMBERS(show_hc_detail_const, hc_memb) = True
 					HEALTH_CARE_MEMBERS(HC_major_prog_const, hc_memb) = "MA"
+					If hc_status_code_past_4_months = False Then HEALTH_CARE_MEMBERS(HC_eval_process_const, hc_memb) = "Recertification"
```
Comment (C-Love) [inline notes/health-care-evaluation.vbs:4988]: I think this is supposed to be "If hc_status_code_past_4_months = True Then ". The logic to get the boolean set correctly, but the 'Recertification' does not come up.

**Review by C-Love — CHANGES_REQUESTED** [review]
The logic is solid and easy to follow. I think you just have the wrong Boolean value set in setting the array parameter for the person.

File: notes/health-care-evaluation.vbs (hunk 1 above)
Comment (MDR22) [reply notes/health-care-evaluation.vbs]: Yes, you are 100% correct. Fixed that.

**Review by C-Love — APPROVED** [review]
Looks great!

**Pull request merged**